### DISCUSSION TITLE
[M-Z] (22 Gennaio 2019) Esercizio 5

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#436] [Esami & Esoneri M-Z]
## Traccia

<img width="876" alt="image" src="https://github.com/user-attachments/assets/c4c1d34a-df30-45cb-9a07-62dce5f57896">

>**Stesso esercizio:** #427 (15 Gennaio 2018 - Esercizio 5)

**Comment by luckignolo32** [comment]
I passaggi algebrici dovrebbero essere corretti, suppongo di sì dato che mi viene...
<img width="648" alt="{05DDD00B-BCA0-413A-B0B8-CFDDA2013191}" src="https://github.com/user-attachments/assets/a61d536d-efd8-4c90-8363-25d5d527894b">
<img width="899" alt="{D1DF8CFA-C581-45E6-9797-226E8DBB52CA}" src="https://github.com/user-attachments/assets/b8f700d3-0741-42d1-aee1-ed5f4a7aac60">

**Comment by rimaout** [comment]
![image](https://github.com/user-attachments/assets/fa160152-8b63-4c7d-9124-7d03748bfa00)

**Comment by matypist** [comment]
Questo esercizio è un duplicato di [[M-Z] (15 Gennaio 2018) Esercizio 5 #427](https://github.com/sapienzastudentsnetwork/metodi-matematici-per-l-informatica/discussions/427), per evitare una frammentazione delle proposte di soluzione potrete quindi scrivere/leggere future eventuali possibili soluzioni direttamente lì
